### ISSUE TITLE
Colorize output

### ISSUE THREAD

**Issue by nikolay** [yokattana/json-yaml#3]
It would be nice to have an option to colorize YAML like [jq](https://stedolan.github.io/jq/) does with JSON. It's very useful for local development.

**Comment by sjmulder** [comment]
I agree, that would be useful. However, I feel that including that feature in json-yaml would violate the single responsibility principle that I like to follow with Unix programs. It should be a different program. Then you can colorise YAML output form any source.
